app/presenters/check_answers/base.rb

```diff
@@ -24,8 +24,8 @@ def row_data
     end
 
     def row_content(head_key, text, head_opts = {}, footer: false)
-      translated_heading = translate_table_key(section, head_key, **head_opts)
-      heading = translated_heading.start_with?('Translation missing:') ? head_key : translated_heading
+      heading = translate_table_key(section, head_key, **head_opts)
+      # heading = translated_heading.start_with?('Translation missing:') ? head_key : translated_heading
```

Comment:
Any reason we are commenting this out?

Reply:
Sorry, I did that as part of some refactoring I did as part of my original approach and then abandoned.. that being said we should have translations for everything and not be doing this...

I left it commented so I would remember to check and ensure removing it didn't break anything... and then forgot about it.. :(

Reply:
have put this back in with a todo to look at it later...